packages/back-end/src/models/SavedGroupModel.ts

```diff
@@ -0,0 +1,79 @@
+import mongoose from "mongoose";
+import { SavedGroupInterface } from "../../types/saved-group";
+import uniqid from "uniqid";
+
+const savedGroupSchema = new mongoose.Schema({
+  id: {
+    type: String,
+    unique: true,
+  },
+  orgId: {
+    type: String,
+    index: true,
+  },
+  groupName: String,
+  owner: String,
+  dateCreated: Date,
+  dateUpdated: Date,
+  values: [String],
+  attributeKey: String,
+});
+
+type SavedGroupDocument = mongoose.Document & SavedGroupInterface;
+
+const SavedGroupModel = mongoose.model<SavedGroupDocument>(
+  "savedGroup",
+  savedGroupSchema
+);
+
+function parseSaveGroupString(list: string) {
+  const listArr = list.split(",");
+
+  const savedGroup = listArr.map((i: string) => {
+    return i.trim();
+  });
+
+  return [
+    ...new Set(savedGroup.filter((value) => value !== "," && value !== "")),
+  ];
+}
+
+type SavedGroupEditableProps = Omit<
+  SavedGroupInterface,
+  "dateCreated" | "dateUpdated" | "id" | "values"
+>;
```

Comment:
Creating the type here is good.  I would separate out the Create props and Edit props since they are going to be different.  For example, you don't want to be able to edit the organization.